faasmcli/faasmcli/tasks/dev.py

```diff
@@ -18,7 +18,9 @@
 
 
 @task
-def cmake(ctx, clean=False, build="Debug", native=False, perf=False):
+def cmake(
+    ctx, clean=False, build="Debug", native=False, perf=False, prof=False
+):
```

Comment:
This was the change I mentioned for supporting local print debugging of performance.